libsqlite3-sys/upgrade.sh

```diff
@@ -17,7 +17,7 @@ export SQLITE3_INCLUDE_DIR=$SQLITE3_LIB_DIR
 cargo update
 # Just to make sure there is only one bindgen.rs file in target dir
 find $SCRIPT_DIR/../target -type f -name bindgen.rs -exec rm {} \;
-cargo build --features "buildtime_bindgen" --no-default-features
+cargo build --features "buildtime_bindgen session unlock_notify preupdate_hook" --no-default-features
```

Comment:
I would prefer not have `session unlock_notify preupdate_hook` activated by default (like SQLite).

Reply:
The only one of these that actually matters for the build is `session`, so the other two are fine.

That said, without this `cargo build --features "session bundled"` fails to compile. Having these declarations in the bindings doesn't actually mean you need SQLite to have session support -- it works fine if sqlite is compiled without session support (you just get a linker error if you call one of these functions).

That said, if you won't change your mind it I could also output both prebuilt bindings with and without session, and choose which one to include in the build.rs. This seems kind of pointless to me, but would work.

Reply:
Eh, I'll just make two binding files. It might be confusing the other way for people who depend on libsqlite3-sys directly

Reply:
Okay, there are two binding files now. That's kind of annoying but could be worse.